File: week-02/p1.go
package main

import (
	"fmt"
	"math"
)

func example_tn_n(n int) int {
	total := 0
	for i := 0; i < n; i++ {
		for j := 0; j < 999; j++ {
			total++
		}
	}
	k := int(math.Sqrt(float64(n)))
	for j := 0; j < k; j++ {
		total++
	}
	return total
}

func main() {
	fmt.Println(example_tn_n(1000))
}

File: week-02/p1.py
def example_t_n_v2(n: int) -> int:
    total = 0
    for i in range(n):
        for _ in range(999):
            total += 1
    k = int(n ** 0.5)
    for _ in range(k):
        total += 1
    return total

print(example_t_n_v2(1000))